workflow-dev/tpl/storage.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 store=`grep "storage\s*=\s*" $HELM_GENERATE_DIR/tpl/generate_params.toml | cut -d "=" -f2 | tr -dc '[:alnum:]\n\r'`
-if [ -n "$STORAGE_TYPE" ]; then
-  if [ "$STORAGE_TYPE" != "minio" ]; then
-    rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
-  fi
+if [ -n "$STORAGE_TYPE" ] && [ "$STORAGE_TYPE" != "minio" ]; then
+  rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
 elif [ "$store" != "minio" ]; then
   rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
 else
```

Comment:
I see what you're saying... as long as the toml has not been edited, that first line is _always_ going to resolve as true, no matter what the env var is set to.

This will work:

```
if [ "$STORAGE_TYPE" = "minio" ] || ( [ -z "$STORAGE_TYPE" ] && [ "$store" = "minio" ] ); then
  helmc tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifests/deis-minio-rc.yaml $HELM_GENERATE_DIR/tpl/deis-minio-rc.yaml
  cp $HELM_GENERATE_DIR/tpl/deis-minio-service* $HELM_GENERATE_DIR/manifests/
else
  rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
fi
```